.gitignore

```diff
@@ -25,3 +25,4 @@ build
 .mypy_cache
 *.whl
 venv
+/.vs
```

Comment:
was this deliberate?

Reply:
Visual Studio added it, and I thought since .vscode is already in there I can include it